OpenBudgeteer.Core/ViewModels/Helper/TransactionListingViewModel.cs

```diff
@@ -84,10 +84,14 @@ public async Task<ViewModelOperationResult> LoadDataAsync(Guid bucketId, bool wi
         {
             _transactions.Clear();
 
+            var allAccounts = ServiceManager.AccountService.GetAll()
+                .Select(i => AccountViewModel.CreateFromAccount(ServiceManager, i)).ToList();
+            var allBuckets = ServiceManager.BucketService.GetAllWithSystemBuckets();
+
             // Get all BankTransaction
             var transactionTasks = ServiceManager.BudgetedTransactionService.GetAllFromBucket(bucketId)
                 .Select(i => 
-                    TransactionViewModel.CreateFromTransactionWithoutBucketsAsync(ServiceManager, i.Transaction))
+                    TransactionViewModel.CreateFromTransactionAsync(ServiceManager, allAccounts, allBuckets, i.Transaction))
```

Comment:
Something for myself after merging this: Consider refactoring of `CreateFromTransactionAsync()` using `Account` instead of `AccountViewModel`